applications/ShapeOptimizationApplication/tests/opt_process_vertex_morphing_test/run_test_small.py

```diff
@@ -74,7 +74,7 @@
 
 # Testing of design output
 output_file_name = os.path.join(output_directory, "design_surface_0_5.vtk")
-reference_file_name = os.path.join(os.path.join(os.path.dirname(os.path.abspath(__file__))), "ref_design_surface_0_5.vtk")
+reference_file_name = os.path.join(os.path.dirname(os.path.abspath(__file__)), "ref_design_surface_0_5.vtk")
```

Comment:
FYI the [pathlib](https://docs.python.org/3/library/pathlib.html) library might make life easier in those situations